### ISSUE TITLE
Option to toggle baritone disconnect in clickgui

### ISSUE THREAD

**Issue by LeafHacker** [ImpactDevelopment/ImpactClient#1016]
Baritone supports disconnecting on arrival, Auto Disconnect could have an option that controls whether or not this is enabled.

**Comment by leijurv** [comment]
Note that this is currently broken in baritone 1.2.5 whoops

But doesn't matter since I don't think impact would respect it anyway?

**Comment by LeafHacker** [comment]
> impact would respect it anyway?

this isn't intended as a functional change, simply a convenient interface for toggling the baritone option (via a dummy impact option)

**Comment by leijurv** [comment]
Okay, this should actually be a functional change. Impact's autoreconnect fires even with baritone's disconnect on arrival, so there should be some feature meme to detect that and respect it idk

**Comment by LeafHacker** [comment]
Yes but that's a separate issue. This is about controling a baritone feature from Disconnect, that is about handling a baritone feature in Reconnect

**Comment by leijurv** [comment]
le sigh ok

**Comment by leijurv** [comment]
ill make another issue

**Comment by leijurv** [comment]
#1036
